testing/trino-tests/src/test/java/io/trino/execution/TestEventListenerBasic.java

```diff
@@ -137,11 +139,13 @@ public Iterable<ConnectorFactory> getConnectorFactories()
                         })
                         .withGetMaterializedViews((connectorSession, prefix) -> {
                             ConnectorMaterializedViewDefinition definition = new ConnectorMaterializedViewDefinition(
-                                    "SELECT nationkey AS test_column FROM tpch.tiny.nation",
-                                    Optional.empty(),
+                                    "SELECT test_varchar, test_bigint FROM mock.default.tests_table",
```

Comment:
why the change here?

Reply:
The cross-connector materialized view has not been supported. However, it was not checked during MV creation but was checked during refresh, which made inconsistent error message with new implementation. Anyway, even in previous implemenation, materialized view was not usable at all because refresh was not allowed. Therefore, I moved the checking logic to MV creation part in the following change, and it revealed the caveat of existing test. (MV creation in mock connector while source table is in tpch connector) That's why I need to change this test to use source table in mock catalog.

https://github.com/trinodb/trino/pull/12570/files/7ca0ce2d2f0ba436eb97e635fbd1d066fa81bb5a#diff-187dbbff78f9bf3e52159fc4cb0e84b971a0ef4f2b2bf6e897efa0a25a70aff8R1243